content/chainguard/administration/network-requirements.md

```diff
@@ -1,29 +1,29 @@
 ---
 title: "Network Requirements"
 linktitle: "Network Requirements"
-lead: "Using Chainguard Images with firewalls, access control lists, and proxies"
+lead: "Using Chainguard Containers with firewalls, access control lists, and proxies"
 type: "article"
-description: "Using Chainguard Images with firewalls, access control lists, and proxies"
+description: "Using Chainguard Containers with firewalls, access control lists, and proxies"
 date: 2023-09-08T08:49:31+00:00
 lastmod: 2024-11-20T15:22:20+01:00
 draft: false
-tags: ["Chainguard Images", "Product", "Reference"]
+tags: ["CHAINGUARD IMAGES", "PRODUCT", "REFERENCE"]
```

Comment:
We should not be hard coding fully uppercased words, if we want to make this a design decision it should be addressed programmatically.